vips/vip-213/vip-213.ts

```diff
@@ -0,0 +1,95 @@
+import { ethers } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const XVS_BRIDGE_ADMIN = "0x70d644877b7b73800E9073BCFCE981eAaB6Dbc21";
+export const XVS_BRIDGE_SRC = "0xf8F46791E3dB29a029Ec6c9d946226f3c613e854";
+export const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+export const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+export const ETHEREUM_TREASURY = "0xFD9B071168bC27DBE16406eC3Aba050Ce8Eb22FA";
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+export const BNB_TREASURY = "0xf322942f644a996a617bd29c16bd7d231d9f35e9";
+export const COMMUNITY_WALLET = "0xc444949e0054A23c44Fc45789738bdF64aed2391";
+
+export const RECEIVER_ADDRESS = ethers.utils.defaultAbiCoder.encode(["address"], [ETHEREUM_TREASURY]);
+export const ADAPTER_PARAMS = ethers.utils.solidityPack(["uint16", "uint256"], [1, 300000]);
+export const XVS_AMOUNT = parseUnits("435000", 18);
```

Comment:
Let's split the XVS transfer in two VIP's:

* in the first one (this one), we will bridge 75,750 XVS
* in the second one, we'll bridge 171,500 XVS

The idea is to cover the Q1 rewards, with these two VIP's